test/fixtures/bn.json

```diff
@@ -0,0 +1,105 @@
+{
+  "valid": [
+    {
+      "dec": "0",
+      "hex": ""
```

Comment:
Fixed the test @indutny, after some further research and thinking about it.  I'm happy with a consistent padding of at least `1`.  
Conceptually I could understand an empty string for zero,  as the absence of any information, but, in this case it wasn't consistent with the decimal result so whatever.